src/common-types/Lang.ts

```diff
@@ -0,0 +1 @@
+export type Lang = 'eng' | 'rus' | 'byn';
```

Comment:
Can we change code for Belarusian language?  ISO code for Belarusian is 'bel' or 'be'

Reply:
That's a good point, but let's do it after. Since it's not the only place where the language is used